.github/actions/cdn_code_deploy/action.yaml

```
@@ -0,0 +1,100 @@
+name: CDN Code Deploy 
+description: An action Deploy the artifact to the CDN
```

```suggestion
description: An action Deploy an artifact to a CDN
```

can you add some details about this artifact? ie. must be a zip archive?